src/main/java/VASSAL/build/module/GlobalOptions.java

```diff
@@ -181,6 +184,21 @@ public void propertyChange(PropertyChangeEvent e) {
     classicMfd.addPropertyChangeListener( (evt) -> setUseClassicMoveFixedDistance(classicMfd.getValueBoolean()));
     prefs.addOption(classicMfd);
 
+    //BR// Drag Threshold
+    final IntConfigurer dragThresholdConf = new IntConfigurer(
+      DRAG_THRESHOLD,
+      Resources.getString("Mouse Drag Threshold"),  //$NON-NLS-1$
+      10
+    );
+    dragThresholdConf.addPropertyChangeListener(new PropertyChangeListener() {
+      @Override
+      public void propertyChange(PropertyChangeEvent e) {
+        dragThreshold = dragThresholdConf.getIntValue(10);
+        System.setProperty("awt.dnd.drag.threshold", Integer.toString(dragThreshold));
+      }
+    });
+    prefs.addOption(dragThresholdConf);
+    
```

Comment:
I'm a bit dubious about adding a preference for this. I can't come up with another program where the drag threshold is settable. How does everything else manage to get a sane value for a drag threshold without user intervention?

Reply:
Judgment call I guess. Of course we're a program that, at heart, lets our users design interfaces for other users, so that's a truly more "meta" situation than many programs. My ultimate profession has always been interface designer, so on the one hand no interface is ever good enough (I can begin to imagine the misery of going through the world with perfect pitch), but on the other hand no tool given to an interface designer ever goes to waste. (In a way, I would never have become involved in Vassal if the interface of the Paths of Glory module my friend talked me into playing last November didn't make me want to stab myself in the eye -- I spent the first two weeks telling myself "I am NOT fucking FIXING this. I am NOT fucking FIXING this." and then, you know, I decided to fix just this one thing, etc, etc, and the rest is history). So I have lots of professional history to say of course you want to give interface designers micro-control of interfaces, but in the end it's all an art form it's not like I've collected data that show it's better to have preferences. I just personally want them so I thought I'd, you know, be the change I want in the world :-)

Reply:
An example use-case would be a card game (e.g. Innovation) where you want the player to have to drag the card a more substantial percentage of the card's size (than 10 pixels) before it "engages".